core/src/main/java/com/digitalpebble/stormcrawler/util/Configurable.java

```diff
@@ -42,7 +45,7 @@ default void configure(
      *     information.
      */
     @NotNull
-    static <T extends Configurable> List<@NotNull T> createConfiguredInstance(
+    static <T extends AbstractConfigurable> List<@NotNull T> createConfiguredInstance(
```

Comment:
Do we need these changes here? The `getName()` is `Configurable`, so is there a real need to change it here?

Reply:
Yes. I opened it in my IDE. Way better than using the web diff view, so yeah - you are right.